src/legacy/ui/public/vis/editors/default/components/agg_params.tsx

```diff
@@ -230,6 +230,7 @@ function DefaultEditorAggParams({
         <EuiFormRow>
           <EuiAccordion
             id="advancedAccordion"
+            data-test-subj={`advancedParams-${agg.id}`}
```

Comment:
If you use the id here you can use `toggleAccordion` from the visualize page object